### ISSUE TITLE
UI issue with player list scrolling

### ISSUE THREAD

**Issue by Bl4cktalon** [itsamewayu/OLFallenDoll#1035]
Currently in order to scroll through the list of players in the lobby using mouse-wheel the mouse cursor must be located over the elements allowing the name list scrolling - that is either the scroll bar at the right from the list or within one of the frames with the player names. The scroll bar is rather narrow object to place the cursor on without focusing, while placing the cursor on a frame with a player name and trying to scroll usually results with the cursor ending in the gap between player name frames and there the mouse scrolling operates the camera zoom instead of list scrolling.
https://gyazo.com/63af6b8435068c1cd9f6af89e58a1e17

tldr:

1) place the cursor on a a frame with player name in the player list and scroll with the mouse-wheel trying to scroll trough the list
2) the cursor ends up in the gap between player names and now you're scrolling the camera zoom instead of the list

Make scrolling in the entire player list area to operate the player list scrolling.
